modules/hostap/Kconfig

```diff
@@ -234,10 +235,26 @@ config EAP_SIM
 config EAP_AKA
 	bool "EAP-AKA support"
 
+config EAP_FAST
+	bool "EAP-FAST support"
+
 config EAP_ALL
 	bool "All EAP methods support"
 	select EAP_TLS
-	default y if WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
+	select EAP_PEAP
+	select EAP_GTC
+	select EAP_TTLS
+	select EAP_MD5
+	select EAP_MSCHAPV2
+	select EAP_LEAP
+	select EAP_PSK
+	select EAP_FAST
+	select EAP_PAX
+	select EAP_SAKE
+	select EAP_GPSK
+	select EAP_PWD
+	select EAP_EKE
+	select EAP_IKEV2
```

Comment:
these should be limited to what is currently supported in Zephyr, like TLS/TTLS/PEAP/GTC.

Reply:
Okay, also add EAP_FAST, which will upstream soon.